mls_validation_service/src/handlers.rs

```diff
@@ -64,13 +64,11 @@ impl ValidationApi for ValidationService {
                 match validate_group_message(message.group_message_bytes_tls_serialized) {
                     Ok(res) => ValidateGroupMessageValidationResponse {
                         group_id: res.group_id,
-                        epoch: res.epoch,
```

Comment:
Epoch was removed from the protos previously, but was still floating around here